force_wfmanager/cli/tests/test_execution.py

```diff
@@ -0,0 +1,31 @@
+import unittest
+import subprocess
+
+
+class TestExecution(unittest.TestCase):
+
+    def test_command_line_invocation(self):
+        """Test if the workflow manager can be loaded from the command line"""
+
+        # Does the command exist?
+        available_commands = subprocess.check_output(['/bin/bash', '-c', 'compgen -c'])
+        available_commands = available_commands.splitlines()
+        command_found = False
+
+        # available_commands is a list with elements of type: 'bytes'
+        for command in available_commands:
+            if command == b'force_wfmanager':
+                command_found = True
+        if command_found is False:
+            self.fail("Command \'force_wfmanager\' does not exist in the list of available bash commands")
+
+        # Does it execute correctly?
+        print("Close the workflow manager to complete the test!")
```

Comment:
do not put print statements in tests, your tests should have nothing on stdout/stderr.

Reply:
This will also hang with the automated CI. Tests should not require manual intervention. They should be fully automatic and report if it's ok or not.

Reply:
You have two strategies for this. Either you invoke the application with something that exits, like a --help or --version option (which we might need) or you kill the process after a given timeout and then take the timeout exception as a signal that everything is fine (because the application started)

Reply:
Additionally, the user won't see the text that is printed because most unit testing frameworks capture standard out and don't display it on the screen unless something goes wrong (or maybe in a very verbose mode).